Add minutes, update readme (#50)
mdn/community-meetings: 2024-05-29/minutes.md

[Florian] And something like proper page templates to to automatically add things like web compat, like browser compat and specification data.

[Florian] Is within, within reach. So. I hope and of the year. All of your life gets.

## Stalled PR discussion (@CBID2)

[Docs: making information about read-only clearer](https://github.com/mdn/content/pull/33193)

[Vadim] Thank you. Okay, we can give a chance to discuss this pull request. Christine if you're still up for it go ahead and unute yourself and we can we can talk about it or we can we can go to Discord.

[Christine] Hold on, let me just, let me just try and find it.

[Vadim] If anyone missed any parts of this this call, we're, we're, life-captioning this and, we're not gonna publish the full transcript, but we're gonna. Post some minutes. In the of this call later and the same repository.

> [Christine] Alright, I just posted it. It's mostly that I need the update.
> 
> [Christine] One of the maintainer suggested, but I never really. Got notified. We'll follow up feedback.
> 
> [Christine] Even though I am tagged her. So.
> 
> [Christine] That's what it mostly is.

Reviewer: ```suggestion
[Christine] It's mostly that I need an update. I've added what the maintainer suggested, I'm waiting for feedback.
```

> [Vadim] Hey, yeah, yeah, we will have a look at it and I see that Estelle is your viewing us and that you were talking on sometime like in April.
> 
> [Vadim] Yeah, I guess you can you can also, tag her again or, We can, some other people from commutes can have a look at this PR.

Reviewer: ```suggestion
[Vadim] I see that Estelle is reviewing this. You can tag her again or some other people from the community can have a look at this PR.
```

> [Christine] Yeah. Alright.
> 
> [Vadim] Okay, thank you for bringing this up. You can, like I mentioned before, you can always, go to to our Discord and then ask for help there or just talk some from content team.
> 
> [Christine] Okay.
> 
> [Vadim] Someone from contributors will be there to help you.

Reviewer: ```suggestion
[Christine] Alright.

[Vadim] Thank you for bringing this up. Like I mentioned before, you can always go to to our Discord and ask for help there

[Christine] Okay.
```

> [Ruth] Yeah, thank you, Christine. We, we can take a look at this one. If you have any in the future that get stools, just ping us in Discord.
> 
> [Vadim] Yep.
> 
> [Ruth] And we can pick them up. Alright, sorry this one has got stalled.
> 
> [Christine] Oh, sure, sure.

Reviewer: ```suggestion
[Ruth] Yeah, thank you, Christine. We can take a look at this one. If you have any in the future that get stalled, just ping us in Discord. Sorry this one has got stalled.
```

> [Vadim] Good. And, that's it for today.
> 
> [Vadim] Yep, I guess we are. And, we're gonna be, we'll think about the next date for the community call, but it most likely gonna be end of June.
> 
> [Vadim] Look for announcement in a Discord or in the discussions we'll let you know once we have the final date.
> 
> [Vadim] And, we'll see each other. In June or in pull requests, you know.
> 
> [Vadim] See you around.

Reviewer: ```suggestion
[Vadim] Good. And, that's it for today. We'll announce the next date for the community call, but it most likely will be around end of June. Look for announcement in Discord or in the GitHub Discussions. 

We'll see each other in June or in pull requests. See you around.
```
